messages/Makefile

```diff
@@ -4,8 +4,36 @@ include default.mk
 ifndef ALPINE
 # Rebuild the docs unless we're already running in the build docker image (running
 # docker from docker doesn't work).
-default: messages.md
+default: .validated messages.md
+else
+default: .validated
 endif
 
+.validated:
+	cd jsonschema && npx ajv compile --spec=draft2020 --strict=true \
+		-s "Source.json" \
```

Comment:
We should generate this list automatically. Fix coming....

Reply:
I realised the order is significant, so a fully automated list using `find` won't work until ajv fixes https://github.com/ajv-validator/ajv-cli/issues/43